### ISSUE TITLE
Suggestion - Main Atribute Stats 

### ISSUE THREAD

**Issue by drAkeEngine** [Indyuce/mmocore#22]
Is it possible that the quantity of an attribute can be used as the basis of a statistic for the classes?

for example if a player has 30 points in the strength attribute
have your class rise 30 attack points

this could be quite useful for different classes to raise the base attack according to the attribute to which they are put as the main
I expose the case

Warrior - Main Attribute: Strength
Magician - Main Attribute: Intelligence
killer- Main Attribute: Dexterity
```
-------------------------------------------------- -------------
Warrior-Class-File
attributes:
     attack-damage:
         base: "Attributelvl: Strenght"
         per-level: 0
-------------------------------------------------- -------------
Magician-Class-File
attributes:
     attack-damage:
         base: "Attributelvl: Intelligence"
         per-level: 0
-------------------------------------------------- -------------
killer-Class-File
attributes:
     attack-damage:
         base: "Attributelvl: Dexterity"
         per-level: 0
```

**Comment by Indyuce** [comment]
You mean having attribute stat bonuses which depend on the player's current class. That could be implemented however I don't feel like it would be that useful. The point of the attribute system is to have something that is the same for all the available classes and to have players select what attribute they want to increase based on the class they gave.

**Comment by drAkeEngine** [comment]
not necessarily, the idea that I propose is that a type of tags can be placed that return a specific value, for example the points that an attribute has and to be able to place it in some statistic within the class configuration.

{attribute: Strenght} this would return the amount of points the player has in that attribute, and could be placed in some stat within the configuration.

attributes:
     attack-damage:
         base: "Attribute: Strenght"
         per-level: 0

This I would like to implement perhaps in the future in conjunction with mmoitems, if for example an item gives you as a bonus 20 Strength points, that the Warriors can have 20 damage points, this can be placed in the current system of one or another shape.

but if those same 20 strength points have a magician, it doesn't make much sense since if the strength attribute gives the damage, you are a warrior wizard or any other class all of them would increase strength or look for items with that attribute.

Thus we would not change the attributes, if we would not leave in the configuration defined in the "attack-damage" section where it can obtain the value or any other characteristic can be configured in this way.

Srry for my bad english
